T.Pizza's Dev Exercises

tpizza opened the pull request:
Fun exercise!

I look forward to speaking soon!

Best,

Theo

tpizza committed:
Completed Leap exercise

tpizza committed:
completed phone number exercise

tpizza committed:
base project commit:
-conversion to BrowserSync for real-time x-platform testing
-added gulp build sys for project build automation
-established sass project
-setup haml views
-created Ruby script for mimicking the Rails render function in gulp-ruby-haml

tpizza committed:
completed CSS build

tpizza committed:
Compiled production-ready build

tpizza committed:
documented the readme

tpizza committed:
refactored styling names in accordance with the GoNoodle style guide

tpizza committed:
refactored animations by migrating from GSAP to CSS3 only

tpizza committed:
created a JS animation fallback for IE9

tpizza committed:
added local copy of TweenMax

tpizza committed:
fixed the absolute pathing on the local TweenMax asset

tpizza committed:
updated leap script to throw type error during year validation

updated CSS test to use js- prefixed classes as hooks in the tile script per style guide

dcalhoun commented:
Thanks for the submission, Theo. Did you intend to include a "gulpfile" within your "css" directory? Executing `npm start` fails for me currently.

tpizza committed:
Added that pesky missing gulpfile

tpizza commented:
Hey @dcalhoun good catch!  Just added and pushed up the missing gulpfile.

tpizza committed:
Automation Infrastructure Update
- Added Bower support
- Added Bundler support
- Cleaned up gulpfile logic

tpizza committed:
Build automation update
- Automated output Compass output styling relative to git branch
- Fixed bug preventing BrowserSync from streaming Compass output updates
- Added Gulp preprocessing to markup to automate scripting dev
- Fixed a syntax error in Typekit's advanced snippet

tpizza committed:
README update as well as added CSS maps
